src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -644,8 +644,8 @@ public virtual IEnumerable<CallStackFrame> GetCallStack()
         /// Adds the provided set of breakpoints to the debugger.
         /// </summary>
         /// <param name="breakpoints">Breakpoints.</param>
-        /// <param name="runspaceId">The runspace id of the runspace you want to interact with, null being the current runspace.</param>
-        public virtual void SetBreakpoints(IEnumerable<Breakpoint> breakpoints, int? runspaceId) =>  
+        /// <param name="runspaceId">The runspace id of the runspace you want to interact with. Defaults to null (current runspace).</param>
+        public virtual void SetBreakpoints(IEnumerable<Breakpoint> breakpoints, int? runspaceId = null) =>
```

Comment:
It is not formatting change - please revert.

Reply:
Good catch, this looks like incorrect merge

Reply:
Yes, this was from incorrect merge, I will rewrite commit.